sections/02_intro.md

```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
 What if, more generally, deep learning could solve the challenges
 presented by the growth of data in biomedicine? Could these algorithms
 identify the "cats" hidden in our data - the patterns unknown to the
-researcher - and act on them? Deep learning has transformed image analysis, but
-what about biomedicine more broadly? In this review,
+researcher - and act on them? In this review,
 we examine whether this transformation is simply a matter of time or
```

Comment:
I tried to edit for this, but we can also bring back the sentence. Let me know what you think about the new version.